### PR TITLE
begin on varbits

### PR DESCRIPTION
**Pull request by Bia10**
add VarbitIndices

### CODE REVIEW

File: src/main/java/rsb/internal/globval/VarbitIndices.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package rsb.internal.globval;
+
+public class VarbitIndices {
+    public static final int SPELL_TO_AUTOCAST = 276; // Permitted values: WidgetIndices.DynamicComponents.AutocastSpells
+    public static final int EQUIPPED_WEAPON_TYPE = 357;
+    public static final int DEFENSIVE_AUTOCAST = 2668;
+    public static final int QUICK_PRAYER = 4103;
+    public static final int CURRENT_BANK_TAB = 4150;
+    public static final int XP_DROPS = 4702;
+    public static final int FAIRY_RING_DIAL_ADCB = 3985; // Left dial
```
Comment (GigiaJ) [inline src/main/java/rsb/internal/globval/VarbitIndices.java:10]: Could this have been 
FAIRY_RING_LEFT_DIAL_ADCB
it'd still be shorter than
FAIRY_RING_LAST_DESTINATION
plus no comments

File: src/main/java/rsb/internal/globval/VarbitIndices.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    public static final int FAIRY_RING_DIAL_ILJK = 3986; // Middle dial
+    public static final int FAIRY_RING_DIAL_PSRQ = 3987; // Right dial
+    public static final int FAIR_RING_LAST_DESTINATION = 5374;
```
Comment (GigiaJ) [inline src/main/java/rsb/internal/globval/VarbitIndices.java:13]: Why not FAIRY_RING_LAST_DESTINATION

File: src/main/java/rsb/internal/globval/VarbitIndices.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    public static final int QUEST_TAB_INDEX = 6590;
```
Comment (GigiaJ) [inline src/main/java/rsb/internal/globval/VarbitIndices.java:14]: QUEST_TAB_INDEX inside a class named VarbitIndices?
If the name is accurate then it is self-explanatory
If not will it need to be changed?

File: src/main/java/rsb/internal/globval/VarbitValues.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    FAIRY_RING_DIAL_K(2),
+    FAIRY_RING_DIAL_L(3),
+    // Fairy Ring Dial PSRQ
+    FAIRY_RING_DIAL_P(0),
+    FAIRY_RING_DIAL_S(1),
+    FAIRY_RING_DIAL_R(2),
+    FAIRY_RING_DIAL_Q(3),
+    // TODO: FAIR_RING_LAST_DESTINATION
+    // Quest Tab Index
+    QUEST_TAB_INDEX_CHARACTER_SUMMARY(0),
```
Comment (GigiaJ) [inline src/main/java/rsb/internal/globval/VarbitValues.java:72]: Is index needed in these names (genuine question)?

**Review by GigiaJ — COMMENTED** [review]
Everything looks great beyond a bit of name weirdness

File: src/main/java/rsb/internal/globval/VarbitIndices.java (hunk 1 above)
Comment (Bia10) [reply src/main/java/rsb/internal/globval/VarbitIndices.java]: agreed, typo fixed

File: src/main/java/rsb/internal/globval/VarbitIndices.java (hunk 2 above)
Comment (Bia10) [reply src/main/java/rsb/internal/globval/VarbitIndices.java]: typo fixed

File: src/main/java/rsb/internal/globval/VarbitValues.java (hunk 4 above)
Comment (Bia10) [reply src/main/java/rsb/internal/globval/VarbitValues.java:72]: no, the index will stay there

**Pull request merged**